### PR TITLE
Colemak keyboard

### PR DESCRIPTION
**Pull request by skyspy20**
Colemak keyboard layout

### CODE REVIEW

File: colemak/colemak_mapping.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+finger1 = [49, 113, 97, 122, 126, 96, 33, 81, 65, 90];
```
Comment (hiamandeep) [inline colemak/colemak_mapping.txt]: seems like you haven't changed mappings. https://colemak.com/wiki/images/e/ef/Colemak_fingers.png

Comment (skyspy20) [reply colemak/colemak_mapping.txt]: Hello, sorry for that i will send you the another pull request with the changes and the layout for the qwertz keyboard in the next couple of hours

File: colemak/colemak_layout.html [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+			<li class="key key-48  keyl-41  dual fin-10 num0">)<br>0</li>
+			<li class="key key-45  keyl-95  dual fin-10 dash">_<br>-</li>
+			<li class="key key-61  keyl-43  dual fin-10 equal">+<br>=</li>
+			<li class="key key-8   keyd-def backspace fin-10">Backspace</li>
+			<!-- 2nd row -->
+			<li class="key key-def keyd-def tab fin-1">Tab</li>
+			<li class="key key-113 keyr-81  letter fin-1 q">Q</li>
+			<li class="key key-119 keyr-87  letter fin-2 w">W</li>
+			<li class="key key-102 keyr-70  letter fin-3 f">F</li>
+			<li class="key key-114 keyr-82  letter fin-4 p">P</li>
```
Comment (hiamandeep) [inline colemak/colemak_layout.html]: correct the ascii values

File: colemak/colemak_layout.html [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+			<li class="key key-110 keyl-78  letter fin-7 n">N</li>
+			<li class="key key-101 keyl-69  letter fin-8 e">E</li>
+			<li class="key key-105 keyl-73  letter fin-9 i">I</li>
+			<li class="key key-111 keyl-79  letter fin-9 o">O</li>
+			<li class="key key-39  keyl-34  fin-10 singleinvertedcomma">"<br>'</li>
+			<li class="key key-10  keyd-def enter fin-10">Enter</li>
+			<!-- 4th row -->
+			<li class="key key-sl  keyd-def shift-left fin-1">Shift</li>
+			<li class="key key-122 keyr-90  letter fin-1 z">Z</li>
+			<li class="key key-120 keyd-88  letter fin-2 x">X</li>
```
Comment (hiamandeep) [inline colemak/colemak_layout.html]: it should be keyr-88 here

File: colemak/colemak_layout.html [hunk 4]
```diff
@@ -33,7 +33,7 @@ <h1>Colemak Keyboard Layout for SpeedCoder</h1>
 			<li class="key key-113 keyr-81  letter fin-1 q">Q</li>
 			<li class="key key-119 keyr-87  letter fin-2 w">W</li>
 			<li class="key key-102 keyr-70  letter fin-3 f">F</li>
-			<li class="key key-114 keyr-82  letter fin-4 p">P</li>
+			<li class="key key-114 keyr-80  letter fin-4 p">P</li>
```
Comment (hiamandeep) [inline colemak/colemak_layout.html]: it should be key-112

File: colemak/colemak_mapping.txt [hunk 5]
```diff
@@ -1,7 +1,7 @@
 finger1 = [96, 126, 49, 33, 81, 113, 8, 65, 97, 90, 122];
 finger2 = [50, 64, 87, 119, 82, 114, 88, 120];
 finger3 = [51, 35, 102, 70, 83, 115, 99, 67];
-finger4 = [52, 36, 53, 37, 114, 80, 103, 71, 116, 84, 100, 68, 118, 86, 98	, 66];
+finger4 = [52, 36, 53, 37, 114, 80, 103, 71, 116, 84, 100, 68, 118, 86, 98, 66];
```
Comment (hiamandeep) [inline colemak/colemak_mapping.txt]: it should be 112 instead of 114

File: colemak/colemak_mapping.txt [hunk 6]
```diff
@@ -0,0 +1,10 @@
+finger1 = [96, 126, 49, 33, 81, 113, 8, 65, 97, 90, 122];
+finger2 = [50, 64, 87, 119, 82, 114, 88, 120];
+finger3 = [51, 35, 102, 70, 83, 115, 99, 67];
+finger4 = [52, 36, 53, 37, 112, 80, 103, 71, 116, 84, 100, 68, 118, 86, 98, 66];
+finger5 = [];
+finger6 = [32];
+finger7 = [54, 94, 55, 38, 74, 106, 76, 108, 72, 104, 78, 110, 75, 107, 77, 109];
+finger8 = [56, 42, 117, 85, 101, 69, 60, 44];
+finger9 = [57, 40, 121, 89, 73, 105, 111, 79, 62, 46];
```
Comment (hiamandeep) [inline colemak/colemak_mapping.txt]: 111 and 79 should be mapped to finger10 instead of finger9

**Pull request merged**